tests/foreman/api/test_contentview.py

```diff
@@ -1032,9 +893,10 @@ def test_cv_associate_rh_custom_spin(self):
         view
 
         """
-        content_view = entities.ContentView(organization=self.org.id).create()
-        content_view.set_repository_ids([self.repo.id])
-        self.assertEqual(len(content_view.read().repository), 1)
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.set_repository_ids([self.repo.id])  # pylint:disable=E1101
+        content_view = content_view.read()
```

Comment:
Ahh, I see what you're asking. Your first code snippet is better than the second because it only incurs one call to the server instead of two calls. As a result, the first bit of code will execute approximately twice as quickly as the second bit of code.